userspace/libsinsp/examples/test.cpp

```diff
@@ -318,12 +353,15 @@ int main(int argc, char** argv)
 		}
 	}
 
-	auto event_names = extract_filter_events(inspector);
-	if(!event_names.empty())
+	auto events_sc_codes = extract_filter_sc_codes(inspector);
+	if(!events_sc_codes.empty())
 	{
-		printf("-- Filter event types names: %s\n", concat_set_in_order(event_names).c_str());
+		auto events_sc_names = libsinsp::events::sc_set_to_sc_names(events_sc_codes);
+		printf("-- Filter AST ppm sc names: %s\n", concat_set_in_order(events_sc_names).c_str());
 	}
```

Comment:
why don't we move this directly inside `open_engine`? in this way, we could hide this ppm_sc patch

Reply:
We set the filter string right before and given this is an example and test binary, leaving it there should increase technical clarity. Unsure if hiding is what we want here.